.github/workflows/add-to-project.yml

```diff
@@ -15,3 +15,13 @@ jobs:
         with:
           project-url: https://github.com/orgs/netdata/projects/32
           github-token: ${{ secrets.NETDATABOT_ORG_GITHUB_TOKEN }}
+
+  add-bugs-to-project:
+    name: Add bugs to project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@v0.1.0
```

Comment:
I think we can just add it as a 2nd step to the first job.

Reply:
thanks for the feedback let me explore that, it is the first time I'm looking to workflows